coriolis_openstack_utils/schemas/instance_assess_info.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "http://cloudbase.it/coriolis-utils/schemas/instance_assessment#",
```

Comment:
Please add `additionalProperties: false` to all items of type 'object' (google it)